Jenkinsfile

```diff
@@ -65,6 +65,9 @@ def gitRepoUrl
 def awsRegion = 'us-east-1'
 def verboseMaven = params.verbose_mvn_logging
 
+//declaring library temporarily
+@Library('bfd@jrosenfeld/BFD-2121-update-global-use-of-awsAuth-for-the-team-to-be-having-more-successful-jenkins-deployments-and-more-velocity') _
+
```

Comment:
This needs to be pulled out prior to approval.

Reply:
Will do. Still resolving a failed build. With the branch name.